website/versioned_docs/version-26.x/ExpectAPI.md

```diff
@@ -69,12 +69,16 @@ test('numeric ranges', () => {
 
 _Note_: In TypeScript, when using `@types/jest` for example, you can declare the new `toBeWithinRange` matcher like this:
 
-```ts
+````ts
```

Comment:
?

Reply:
@SimenB not getting what you are saying? could you elaborate 😊

Reply:
@mrazauskas oh 😅 did not saw that will fix it and push it

Reply:
yeah, the diff has an added backtick here and in a few of the other files (but not all)